packages/blockchain-info-components/src/Form/SelectInput/template.js

```diff
@@ -12,11 +12,28 @@ const StyledSelect = styled(Select)`
   .control {
     box-shadow: none;
     border: 1px solid ${props => props.theme[props.borderColor]};
+    border-left: ${props => props.asideTextInput && 'none'};
```

Comment:
@plondon should these styles be applied elsewhere? feels weird to have all this very situation specific css in the main components repo.

Reply:
absolutely, you can wrap `SelectBox`/`SelectInput` with `styled` and use `.control` selector

Reply:
asideTextInput should be removed in this case

Reply:
You can make your own dropdown by extending this one. This should not be here.